src/content/en/fundamentals/performance/rail.md

```diff
@@ -108,8 +119,7 @@ Working in 50ms blocks allows the task to finish while still ensuring instant re
 
```

Comment:
This section is a bit confusing.

"To allow for <100ms response, the app must yield control back to **the** main thread every <100ms"
Didn't we just say 50ms?

We allow 100 ms of event processing time, and 50 ms of idle processing, which can result in a maximum event handling time of 150ms, if an event shows up right at the beginning of the idle period.

That's a bit subtle though...